### PR TITLE
Fetch and display data from RPA Land API

### PR DESCRIPTION
**Pull request by djbingham**
https://eaflood.atlassian.net/browse/ELM-1552

### CODE REVIEW

File: app/routes/apply/business.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const getBusinessViewModel = require('../../view-models/apply/business')
+const landService = require('../../services/land-service')
+
+module.exports = {
+  method: 'GET',
+  path: '/apply/{sbi}',
+  options: {
+    handler: async (request, h) => {
+      try {
+        const { sbi } = request.params
```
Comment (Matthew-Collins-Defra) [inline app/routes/apply/business.js:10]: SBI is passed through to the page, as well as the api call with no validation other than the default route validation built into Hapi. I couldn't manage to exploit this, but I think the input should be validated by using the validate option in the route config.

File: app/services/land-service.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+const getRpaLandServiceUrl = function (entity, sbi) {
+  return `${config.rpaLandServiceUrl}/${entity}/MapServer/0/query?where=SBI=${sbi}&outFields=*&f=geojson`
+}
+
+const landService = {
+  async getParcels (sbi) {
+    const landParcelsUrl = getRpaLandServiceUrl('LandParcels', sbi)
+
+    const response = await restClient.get(landParcelsUrl)
```
Comment (Matthew-Collins-Defra) [inline app/services/land-service.js:12]: No error checking of the result, which results in the service failing if invalid data is supplied. Not a must fix for this quick throw together, but maybe add a TODO comment to fix this later.

File: app/routes/apply/business.js (hunk 1 above)
Comment (djbingham) [reply app/routes/apply/business.js:12]: You're quite right. I've added validation for SBIs so they must be nine characters, all numeric.

File: app/services/land-service.js (hunk 2 above)
Comment (djbingham) [reply app/services/land-service.js]: Good point. I've added handling for errors calling the API, unexpected response format and responses with no features.

The RPA API doesn't seem to throw errors for non-existent SBIs, it just returns the same data structure with an empty `features` array. Based on that, I've treated an empty `features` array as if it was a "not found" error so we can show some error handling through our front-end.

**Pull request merged**